package.json

```
@@ -35,13 +36,15 @@
   },
   "devDependencies": {
     "browserify": "^16.0.0",
+    "dtslint": "^4.0.5",
```

for consistency it's good to keep the semver range as wide as possible
```suggestion
    "dtslint": "^4.0.0",
```